### ISSUE TITLE
Misaligned accent tag in the training set and classification model

### ISSUE THREAD

**Issue by beckybyte** [WangHelin1997/CapSpeech#3]
Great work. 

I noticed that the accent tag is more fine-grained in [huggingface dataset](https://huggingface.co/datasets/OpenSound/CapTTS-SFT/viewer/default/test?views%5B%5D=test) than the one used in the [dialect classification model](https://github.com/WangHelin1997/CapSpeech/blob/main/capspeech/eval/src/example/dialect_world_dialect.py), is there any mapping from the fine-grained one to the coarse one? If not, what's the test set for the dialect classification? 

Thank you.

**Comment by WangHelin1997** [comment]
Hi, thanks for interest in our work.
Please check this meta data [CapTTS-SFT](https://huggingface.co/datasets/OpenSound/CapTTS-SFT).

**Comment by WangHelin1997** [comment]
Hi, please use the following mapping.

```
dict = {
    "english":"English",
    "southafrican":"South African",
    "canadian":"North America",
    "indian":"South Asia",
    "american":"North America",
    "scottish":"Scottish",
    "british":"English",
    "irish": "Irish",
    "northernirish":"Northern Irish",
    "welsh":"Welsh",
    "australian":"Oceania",
    "newzealand":"Oceania"
}
```
